plugins/dropdown/README.md

```diff
@@ -0,0 +1,98 @@
+# Builder.io example plugin
+
+See [here](src/plugin.tsx) for the React component that powers this plugin
+
+## Status
+
+Builder plugins are in beta. If you run into any issues or have questions please
+contact help@builder.io for help
+
+## Creating a new plugin from this example
+
+### Install
+
+```bash
+git clone https://github.com/BuilderIO/builder.git
+cp -r plugins/example plugins/your-plugin-name
+cd plugins/your-plugin-name
+npm install
```

Comment:
we can remove the clone / getting started instructions here and just say `npm install` to install

Reply:
👍